crates/rslint_parser/test_data/inline/err/jsx_namespace_member_element_name.jsx

```diff
@@ -1 +1,3 @@
+<namespace:a></namespace:a>
 <namespace:a.b></namespace:a.b>
+<dashed-namespace:a.b></dashed-namespace:a.b>
```

Comment:
You need to add semicolons at the end of every statement or it will parse this as binary expressions
```suggestion
<namespace:a></namespace:a>;
<namespace:a.b></namespace:a.b>;
<dashed-namespace:a.b></dashed-namespace:a.b>;
```

Reply:
Done.